Jenkinsfile

```diff
@@ -54,14 +59,12 @@ pipeline {
     stage('Apply Kubernetes files') {
       agent { label 'sbt-template' }
       environment {
-        CASSANDRA = credentials('cassandra-db')
-        CASSANDRA_HOST = 'cluster1-dc1-service.cassandra-operator.svc.cluster.local:9042'
         VAULT_ADDR = credentials('vault-addr')
         VAULT_TOKEN = credentials('vault-token')
-        PDND_INTEROP_KEYS = credentials('pdnd-interop-keys')
-        USER_REGISTRY_API_KEY = credentials('userRegistryApiKey')
-        DOCKER_REPO = 'gateway.interop.pdnd.dev'
-        //REPLICAS_NR = 1
+        INTEROP_KEYS = credentials('pdnd-interop-keys')
+        DOCKER_REPO = "${env.DOCKER_REPO}"
+//         MAIN_AUDIENCE = "" TODO add audience
```

Comment:
Is this TODO intentional (even if commented)

Reply:
Yes, it was intentional, but probably it could be set equal to INTEROP_JWT_AUDIENCE.